src/components/Tools.js

```diff
@@ -27,6 +27,26 @@ class Tools extends Component{
               <Button variant="primary">Portfolios</Button>
             </Card.Body>
           </Card>
+          <Card style={{ width: '18rem' }}>
+            <Card.Img variant="top" src="holder.js/100px180" />
+            <Card.Body>
+              <Card.Title>Goal</Card.Title>
+              <Card.Text>
+                Goals of the student.
+              </Card.Text>
+              <Button variant="primary">Portfolios</Button>
```

Comment:
This button (and the button in the performance section below) still say "Portfolios", though they aren't for portfolios